api/utils.py

```diff
@@ -68,7 +68,7 @@
         CLOB.__visit_name__: CharField,
         DECIMAL.__visit_name__: DecimalField,
         Date.__visit_name__: DateTimeField,
-        Variant.__visit_name__: CharField
+        Variant.__visit_name__: IntegerField
```

Comment:
I agree with using `IntegerField` as the default for `Variant`, assuming that covers Big Integers too.